jestconfig.json

```diff
@@ -3,7 +3,7 @@
     "./jestsetup.js"
   ],
   "moduleDirectories": [
-    "node_modules"
+    "./node_modules"
```

Comment:
Any reason we're using `./` here and `<rootDir>` below? They seem to accomplish the same thing.

Reply:
Nope, no particular reason. in fact wouldn't even really need `./` since jest pwd is ./. Will update.